driver/apdu/pcsc.c

```diff
@@ -104,13 +112,13 @@ static int pcsc_iter_reader(int (*callback)(int index, const char *reader, void
 static int pcsc_open_hCard_iter(int index, const char *reader, void *userdata)
 {
     int ret;
-    int id;
```

Comment:
It changes nothing on Windows. Windows uses LLP64 data model, it means sizeof(int) == sizeof(long) == 4.

Reply:
> Clang-Tidy: Narrowing conversion from 'long' to signed type 'int' is implementation-defined

Reply:
I mean you should use `long long` and `strtoll()`. `long` should be avoid in cross-platform programming. And it may be better to use `unsigned long long` and `strtoull()` since it's an index.